apps/static-site/src/helpers/metadata.ts

```diff
@@ -0,0 +1,44 @@
+import { Metadata } from "next";
+import { getTranslations } from "next-intl/server";
+
+export const siteURL = "https://heritage.tudelft.nl";
```

Comment:
This causes a 404 on the preview branches. Might be fine though, just in case a search indexer indexes a deploy preview - it would not change the canonical URL. The only issue with this deploy preview is the logo link (default og:image) does not load. 

@sammeltassen Could you provide the default image you'd like us to add.

Reply:
this is resolved in the follow up pr #34